tests/models/test_tpu.py

```diff
@@ -23,6 +24,7 @@
 
 
 @pytest.mark.skipif(not TPU_AVAILABLE, reason="test requires TPU machine")
+@dutils.pl_multi_process_test
```

Comment:
when was this function added? and never used till now?

Reply:
i added it a few days ago when the spawn stuff came out...